tests/test_firmware.py

```diff
@@ -12803,6 +12803,40 @@ def read(self, num_bytes):
     assert num_callbacks == 6
 
 
+def test_is_semver_string(mocker, m5stickv, tdata):
+    from krux.firmware import is_semver_string
+
+    assert is_semver_string("0.0.0")
```

Comment:
Could add one `assert is_semver_string("25.03.11")` just to be a more real example